documentation/changelog.rst

```diff
@@ -25,6 +25,7 @@ v0.11.1 | September XX, 2022
 
 Bugfixes
 -----------
+* Do not fail asset page if none of the sensors has any data [see `PR #492 <http://www.github.com/FlexMeasures/flexmeasures/pull/492>`_]
```

Comment:
492 is not a pull request, but a ticket. It's PR is 493. I see that also in #493' changelog you wrote "492".